.github/workflows/test-lint.yml

```diff
@@ -39,6 +39,4 @@ jobs:
           install-mode: goinstall
           args: --timeout 10m
           github-token: ${{ secrets.github_token }}
-          skip-go-installation: true
```

Comment:
We don't need to install Go again since we are using the `actions/setup-go@v5` action

Reply:
Right but those parameters doesn't exists in the action it says

Reply:
ohh I see, the flag was removed

https://github.com/golangci/golangci-lint-action?tab=readme-ov-file#compatibility